staging/com.scoreSmart/device.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const Tp = require('thingpedia');
+const unirest = require("unirest");
+
+module.exports = class ScoreDevice extends Tp.BaseDevice {
+  constructor(engine, state) {
+    super(engine, state);
+    this.uniqueId = 'com.scoreSmart';
+    this.name = 'ScoreSmart';
+    this.description = 'Keep up with the latest in soccer';
+  }
+
+  // getting league standings, given league id
+  get_standings({ league_id }) {
+    const request = require('request');
+
+    const options = {
+      method: 'GET',
+      url: 'https://api-football-v1.p.rapidapi.com/v3/standings',
+      qs: { season: '2020', league: league_id },
+      headers: {
+        'x-rapidapi-key': 'b9b9b1e851msh45c99175d8f8df8p103e18jsn903a05143134',
+        'x-rapidapi-host': 'api-football-v1.p.rapidapi.com',
+        useQueryString: true
+      }
+    };
+
+    request(options, function (error, r, body) {
+      if (error) throw new Error(error);
+      const a = JSON.parse(body);
+      const b = a.response[0].league.standings[0];
+      return b.map((obj) => {
```

Comment:
This return is not right. You're returning inside the async callback. Async callbacks do not return values, the return value is always ignored. You should switch to a promise (async/await) approach, it's cleaner.